buddypress/members/single/edit.php

```diff
@@ -16,6 +16,16 @@
                     print __('Your Account has been created! You can keep adding to your profile or dive right in. You are now ready to connect with other users, participate in events and projects, and get involved in the Mozilla community.', "community-portal");
                 ?>
             </p>
+			<?php if(isset($subscribed) && intval($subscribed) !== 1): ?>	
+				<p class="profile__error-message">
+					<?php 
+						echo __("Notice: We had a problem registering you for our newsletter. Please try signing up again later. To try again ");
+					?>
+						<a class="newsletter__link" href="/newsletter">
```

Comment:
Do we have this page created?

Reply:
@dnaluz I've added the code for the page named "newsletter",  but you would need to create it locally.

Reply:
We would then need to create it in production once this is live.